### PR TITLE
Alpine 3.18.0 and test ruby 3.2

### PR DESCRIPTION
**Pull request by jmthomas**
I don't want to merge and release this quite yet since Alpine 3.18.0 is so new

### CODE REVIEW

File: openc3/Gemfile [hunk 1]
```diff
@@ -11,3 +11,5 @@ gem 'rails', '~> 7.0.0'
 gem 'bootsnap', '>= 1.9.3', require: false
 gem 'rack-cors', '~> 2.0'
 gem 'tzinfo-data'
+# httpclient hasn't been released since 2016 so use Shopify's updated one
+gem 'httpclient', git: 'https://github.com/Shopify/httpclient'
```
Comment (ryanmelt) [inline openc3/Gemfile:15]: Can't do this as it always requires an internet connection.

Comment (ryanmelt) [reply openc3/Gemfile:15]: We probably need to switch to Faraday.

Comment (jmthomas) [reply openc3/Gemfile]: Could also use HTTP: https://github.com/httprb/http

File: openc3/lib/openc3/script/screen.rb [hunk 2]
```diff
@@ -25,7 +25,7 @@ def get_screen_list(scope: $openc3_scope)
         endpoint = "/openc3-api/screens"
         # Pass the name of the ENV variable name where we pull the actual bucket name
         response = $api_server.request('get', endpoint, scope: scope)
-        if response.nil? || response.code != 200
+        if response.nil? || response.status != 200
```
Comment (ryanmelt) [inline openc3/lib/openc3/script/screen.rb:28]: This is breaking to other users (including my current CFDP code).  Can we make code/status aliases (both work?)

**Review by ryanmelt — CHANGES_REQUESTED** [review]

**Pull request merged**